client/src/features/add-bet/components/forms/finish-bet-form/FinishBetButtons.tsx

```
@@ -73,17 +74,17 @@ export const FinishBetButtons = ({
     <>
       {!addStake ? (
         <div className="finish-bet-buttons">
-          <Button
+          <Button2
             type="button"
             onClick={() => setAddStake(true)}
-            className="btn btn-filled"
+            className="btn2-cta"
             children="Add Stake"
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)